apps/meteor/client/components/FilterByText.tsx

```diff
@@ -46,7 +46,7 @@ const FilterByText = ({
 	}, []);
 
 	return (
-		<Box mb='x16' is='form' onSubmit={handleFormSubmit} display='flex' flexDirection={shouldFiltersStack ? 'column' : 'row'} {...props}>
```

Comment:
why did you remove this?

Reply:
We were sending non html properties to the element, which was throwing warnings. Looking into this component's interface there's no valid attribute to be spreaded, so I removed it.